src/test/txpackage_tests.cpp

```diff
@@ -39,6 +39,30 @@ inline CTransactionRef create_placeholder_tx(size_t num_inputs, size_t num_outpu
     return MakeTransactionRef(mtx);
 }
 
+// Context-free check that a package only contains a tx (the last tx in the package) with its
+// ancestors. Not all of the tx's ancestors need to be present.
+bool IsAncestorPackage(const Package& package)
```

Comment:
this should assert that package is non-empty

Reply:
Aha, thanks!